src/tasks/phase2/data_preprocessing.py

```diff
@@ -0,0 +1,64 @@
+from database_connection import DatabaseConnection
```

Comment:
I think it would be better if we can keep data_preprocessing.py in a separate folder in src and we can come up with a better name than "phase2" for that folder. @Thek10patil  would like your input since you would be calling functions in this file?

Reply:
Yes we should put database_connection and data_processing in src folder directly.